tests/package.json

```diff
@@ -13,7 +13,7 @@
   "devDependencies": {
     "@codspeed/vitest-plugin": "^3.1.1",
     "@playwright/test": "1.47.2",
-    "@rsbuild/core": "1.0.7",
+    "@rsbuild/core": "1.0.12",
```

Comment:
Why this not be bumped by renovate

Reply:
![image](https://github.com/user-attachments/assets/137cfe79-1fab-4ecb-92ac-8edbb2b76802)
we should modify this and the `e2e/cases` is deprecated now.